rustls/src/verify.rs

```diff
@@ -15,6 +15,7 @@ use ring::digest::Digest;
 
 use std::sync::Arc;
 use std::time::SystemTime;
+use webpki::CertRevocationList;
```

Comment:
Style nit: should separate the webpki import from the std import block.

Reply:
This fell away when I introduced the local newtype to wrap `Vec<u8>` the ~two places we want to use the webpki type I changed to be explicit references to `webpki::CertRevocationList`.